packages/ui/src/views/user-management/components/page-components/content/components/users-list/users-list.tsx

```diff
@@ -0,0 +1,111 @@
+import { Avatar, Badge, MoreActionsTooltip, SkeletonList, Table } from '@/components'
+import { getInitials } from '@/utils/utils'
+import { DialogLabels } from '@/views/user-management/components/dialogs'
+import { useDialogData } from '@/views/user-management/components/dialogs/hooks'
+import { ErrorState } from '@/views/user-management/components/page-components/content/components/users-list/components/error-state'
+import { NoSearchResults } from '@/views/user-management/components/page-components/content/components/users-list/components/no-search-results'
+import { useSearch } from '@/views/user-management/providers/search-provider'
+import { useStates } from '@/views/user-management/providers/state-provider/hooks/use-states'
+import { useUserManagementStore } from '@/views/user-management/providers/store-provider'
+
+export const UsersList = () => {
+  const { useAdminListUsersStore } = useUserManagementStore()
+
+  const { users } = useAdminListUsersStore()
+  const { searchQuery } = useSearch()
+  const { handleDialogOpen } = useDialogData()
+
+  const { loadingStates, errorStates } = useStates()
+  const { isFetchingUsers } = loadingStates
+  const { fetchUsersError } = errorStates
+
+  if (isFetchingUsers) {
+    return <SkeletonList />
+  }
+
+  if (fetchUsersError) {
+    return <ErrorState />
+  }
+
+  // here should be additional check for users.length === 0,
+  // but until backend is not ready I leave only searchQuery to make it possible to see how this component works
+  // TODO: add additional check for users.length === 0 when backend will be ready
+  if (searchQuery) {
+    return <NoSearchResults />
+  }
+
+  return (
+    <Table.Root variant="asStackedList">
+      <Table.Header className="h-[46px]">
+        <Table.Row className="hover:bg-transparent">
```

Comment:
```suggestion
        <Table.Row className="pointer-event-none">
```

we can use the `pointer-event-none` class